Header header
geometry_msgs/Point32 center
geometry_msgs/Point32 major_axis
geometry_msgs/Point32 major_axis_end
geometry_msgs/Point32 minor_axis
geometry_msgs/Point32 minor_axis_end
